### DISCUSSION TITLE
What am I doing wrong?

### DISCUSSION THREAD

**Discussion by HumanServitor** [imartinez/privateGPT#252] [Q&A]
Installation seemed to go ok. Getting this when I enter a query:

gpt_tokenize: unknown token 'Γ'
gpt_tokenize: unknown token 'Ç'
gpt_tokenize: unknown token 'Ö'
gpt_tokenize: unknown token 'Γ'
gpt_tokenize: unknown token 'Ç'
gpt_tokenize: unknown token 'Ö'

**Comment by MrVolts** [comment]
Same issue

**Comment by NPap0** [comment]
Have encountered the same issue. If you wait a bit you'll notice that the model will get you an output. But its going to take a lot longer than expected. 5-10 minutes depending on your specs. You can also monitor if the script is still running by checking the percentage usage of your cpu and ram in the task manager.

refer to this issue.
https://github.com/imartinez/privateGPT/issues/107

**Reply by HumanServitor** [reply]
Interesting, thanks.

**Reply by thekit** [reply]
I am seeing the same thing.
